restart.sh

```diff
@@ -0,0 +1,16 @@
+
+#!/bin/bash
```

Comment:
```suggestion
#!/bin/bash
```

Make sure `chmod` is right too -- I can't tell just from the web UI.

Reply:
It has execution rights 👍🏼